### PR TITLE
Include caveat re: shinyapps.io and env variables

### PR DESCRIPTION
**Pull request by phobson**
Possibly closes https://github.com/rstudio/rsconnect-python/issues/437

## Intent

(Possibly) Closes https://github.com/rstudio/rsconnect-python/issues/437 via clarification

## Type of Change

- [ ] Bug Fix            
- [ ] New Feature       
- [ ] Breaking Change   
- [x] small doc edit
- [ ] 
## Checklist

- [ ] I have updated [CHANGELOG.md](../CHANGELOG.md) to cover notable changes.
- [ ] I have updated all related GitHub issues to reflect their current state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -433,7 +433,9 @@ filename referenced in the manifest.
 
 ### Environment Variables
 You can set environment variables during deployment. Their names and values will be
-passed to Posit Connect during deployment so you can use them in your code.
+passed to Posit Connect during deployment so you can use them in your code. Note that 
+if you're using `rsconnect` to deploy to shinyapps.io, environment variable management
```
Comment (annikakrilov) [inline README.md:437]:
```suggestion
if you are using `rsconnect` to deploy to shinyapps.io, environment variable management
```

**Review by annikakrilov — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -433,7 +433,9 @@ filename referenced in the manifest.
 
 ### Environment Variables
```
Comment (annikakrilov) [inline README.md:434]:
```suggestion
### Environment variables
```

**Comment by nealrichardson** [comment]
Does shinyapps.io just silently ignore the --environment parameters? Can this error informatively so that you don't have to read the docs to figure out why the app isn't working as expected?

**Comment by mmarchetti** [comment]
> Does shinyapps.io just silently ignore the --environment parameters? Can this error informatively so that you don't have to read the docs to figure out why the app isn't working as expected?

Yes, I'll open a separate PR for that, since this is a user contributed PR.

**Pull request merged**